docs/index.asciidoc

```diff
@@ -110,13 +110,48 @@ append and indirect cannot be combined and will fail validation. +
 ===============================
 
 *Delimiter repetition:* +
-In the source text if a field has variable width padded with delimiters, the padding will be ignored. +
-e.g. for texts of:
+Delimiter repetition in the source text can occur unintentionally or intentionally. +
```

Comment:
I'm wondering if the distinction between intentional vs unintentional repetition is useful here. I mean, if you have a csv with repeated commas to represent blank fields (like in `Jane Doe,4321 Fifth Avenue,,,New York,87432`), I would not describe the repeated delimiters as unintentional. I would just say that they are used to represent empty values. Are there other situations where there might be what you call unintentional delimiter repetition?

If we're talking about the difference between repeated delimiters that are used to express empty values vs delimiters used for padding, I'd focus on that distinction rather than unintentional vs intentional. Then you just need to say something about how the default behavior has changed so that the dissect plugin assumes that repeated delimiters represent blank values rather than padding. Then show how you code the dissect filter if the repeated delimiters represent padding.

To explain the difference, use examples. I'd change the order of how you show the examples. Start out by showing the source text that's being processed, the dissect filter that's being applied, and then the result (what the event looks like). Users expect to be able to scan reference info quickly and rarely read carefully.